app/views/govuk_component/docs/taxonomy_sidebar.yml

```diff
@@ -6,6 +6,10 @@ body: |
 
   Each item is a hash with a title, url, description, and list of related content
   associated with that item.
+accessibility_criteria: |
+  - Should have a role of 'complementary' as it complements the main page content
```

Comment:
I think the role here and in related_items should be navigation, but happy to be argued down.

Reply:
Both components have the `role="complementary"` on the outer container, and then the navigation elements inside each component have `role="navigation"`, so I think it's okay as is.